searx/settings.yml

```diff
@@ -518,6 +518,64 @@ engines:
     shortcut : qws
     categories : social media
 
+    # recoll is a local search engine based on Xapian:
+    # http://www.lesbonscomptes.com/recoll/
+    #
+    # By itself recoll does not offer web or API access,
+    # this can be achieved using recoll-webui:
+    # https://github.com/koniu/recoll-webui
+    #
+    # As recoll-webui by default does not support paged JSON
+    # results it is advisable to use a patched version which does:
+    # https://github.com/Yetangitu/recoll-webui/tree/jsonpage
+    # A pull request was sent upstream, if this is merged the patched
+    # version is no longer needed
+    #
+    # This engine uses a custom 'files' result template
+    #
+    # set base_url to the location where recoll-webui can be reached
+    # set mount_prefix to the location where the file hierarchy is mounted on your _local_ filesystem
+    # set dl_prefix to a location where the file hierarchy as indexed by recoll can be reached
+    # 
+    # For example:
+    #
+    #     Recoll indexes a local filesystem mounted in /export/documents/reference
+    #     The Recoll search inteface can be reached at https://recoll.example.org/
+    #     The contents of this filesystem can be reached though https://download.example.org/reference
+    #
+    #        set base_url to https://recoll.example.org/
+    #        set mount_prefix to /export/documents
+    #        set dl_prefix to https://download.example.org
+    #
+    #     the resulting url will be https://download.example.org/reference
+    # 
+    # set search_dir to the part of the indexed file hierarchy to be searched, use an empty string
+    #     to search the entire search domain
+
+    # this entry (with search_dir set to an empty string) covers the entire recoll search domain
+  - name: library
```

Comment:
Could you comment out these two engines? Engines which require further settings by users should be commented out, so searx can start up using the default config.

Reply:
Done